frontend/src/common/types/app/reducers-state.type.ts

```diff
@@ -0,0 +1,11 @@
+type CounterState = {
+  value: number;
+};
+
+type AuthState = {
+  user: Record<string, unknown>; // TODO: change to User type;
+  isAuthorized: boolean;
+  isLoading: boolean;
```

Comment:
Instead of `isLoading` flat it would be better to use `DataStatus` enum
```
enum DataStatus {
  IDLE = 'idle',
  PENDING = 'pending'
  SUCCESS = 'success',
  ERROR = 'error'
}
```